pkg/eventshub/sender/sender.go

```diff
@@ -179,9 +179,13 @@ func Start(ctx context.Context, logs *eventshub.EventLogs, clientOpts ...Option)
 			return err
 		}
 
+		eventString := ""
```

Comment:
nit, so that it's clear that the request wasn't an event
```suggestion
		eventString := "unknown"
```

Reply:
I like this suggestion, @gabo1208